scripts/model_zoo/update-models-from-caffe2.py

```diff
@@ -217,8 +219,9 @@ def onnx_verify(onnx_model, inputs, ref_outputs):
     'inception_v2': 'inception_v2',
     'resnet50': 'resnet50',
     'squeezenet': 'squeezenet_old',
-    'vgg16': 'vgg16',
+    #'vgg16': 'vgg16',
```

Comment:
Why don't we just remove it?

Reply:
I will add it back soon. I know pytorch and caffe2 both have them. :-)